### ISSUE TITLE
F1 / quick-open-widget are clipped at start

### ISSUE THREAD

**Issue by AndersMad** [Microsoft/monaco-editor#349]
monaco-edito: **v0.8.1**

**repro:** press F1 in playground editor: the labels/rows are -margin 15px displayed/clipped

**reason:** class `quick-open-entry` has child element class `row` - this conflicts with bootstrap v3 since it that  `.row { margin-left: -15px; ... }`

also bump: #70 "The "Command Palette" overlay widget does not display in small (height) editors" 🙏

**Comment by palamccc** [comment]
It even happens in https://microsoft.github.io/monaco-editor/
![monaco editor - mozilla firefox_006](https://cloud.githubusercontent.com/assets/202276/22636509/ceede388-ec61-11e6-8132-3da874d71b85.png)

**Comment by cpetrov** [comment]
The class `row` used by Monaco collides with the Bootstrap class with the same name.

**Comment by olane** [comment]
Getting the same issue, thanks for the workaround.

**Comment by alexandrudima** [comment]
This is not an issue when loading the editor in a clean (no other css) page:

<img width="676" alt="screen shot 2017-06-15 at 11 55 18" src="https://user-images.githubusercontent.com/5047891/27176094-92d47860-51c1-11e7-9f57-3627c7a828ff.png">

I believe the CSS from bootstrap has a class selector for ".row"

**Comment by alexandrudima** [comment]
@joaomoreno We could workaround this on our side by using `monaco-row` or some other class name than `row` on our side in the tree.

**Comment by AndersMad** [comment]
yes, as I said its a bootstrap (only) conflict - but I suggest that all monoco css class names should be prefixed with `monaco-` to avoid any conflicts (and bs should have had bs prefix - but it ain't so)
